json-schema/schema.json

```diff
@@ -76,7 +76,8 @@
           "items": {
             "type": "string",
             "format": "date-time",
-            "pattern": "(\\+00:00|Z)$"
+            "pattern": "(\\+00:00|Z)$",
+            "minItems": 2
```

Comment:
minItems must be used one level up!

Reply:
thanks. Is there a way to expect failures with the current CI setup? That way I would be able to check that the schema actually does what I intend it to do.

Reply:
No, sorry.